lib/control/v1/authenticate.js

```diff
@@ -1,41 +1,39 @@
 'use strict';
 
+const valid = require('../validation/valid');
+
 const HTTP_OK = 200;
 const HTTP_METHOD_NOT_ALLOWED = 405;
+
 const leaseDuration = 300;
+const bodyParser = require('body-parser');
 
 function Authenticate(app) {
-  const route = app.route('/v1/authenticate');
+  const route = app.use(bodyParser.json()).route('/v1/authenticate');
```

Comment:
These are examples several Express anti-patterns that I've been seeing in a number of our projects for a bit now.

* `use` calls, like `use(bodyParser.json())`, should be close to the instantiation of the `app` object, at least in the same module. Otherwise they don't behave deterministically. In this ☝️ case, the JSON body-parser will apply to any routes attached to the `app` _after_ this one, regardless of where they're defined. The JSON parser also has the side-effect of blocking request handling until the `end` event fires, because it needs to aggregate the whole body. Again, this will only be applied to routes attached _after_ this one.

    ```
    const app = Express();
    // ...
    app.use(BodyParser.json());
    ```

* Don't use the `route` method. This seems to be a propagating through other projects and needs to stop. Call VERB methods directly on the `app`. It makes the logic of the controllers much easier to follow visually:

    ```
    app.post('v1/authenticate', middle, ware, handler, functions);
    ```

    If you _really_ want to implement method-not-allowed responses, (1) put the handler into its own module for reuse throughout the project: we have a bunch of these boiler-plate bits knocking around, and (2) define any permitted methods as above ☝️, then put that nice reusable module that you wrote in (1) after them:

    ```
    app.post('v1/authenticate', middle, ware, handler, functions);
    // Any other supported VERBs
    app.all('/v1/authenticate', Handlers.allowed('POST');
    ```